app/models/intervention.rb

```diff
@@ -386,6 +392,10 @@ def printed_on
     printed_at.to_date
   end
 
+  def during_financial_year?
+    FinancialYear.opened.where('? BETWEEN started_on AND stopped_on', printed_at).any?
```

Comment:
Is Between inclusive? If yes, i think its a problem for stopped_on.

Reply:
`BETWEEN` is inclusive on both end.